cmd/resources/resources.go

```diff
@@ -169,6 +190,7 @@ func NewResourceCmd(
 	}
 
 	parentCmd.AddCommand(cmd)
+	parentCmd.Annotations[resourceName] = "resource"
```

Comment:
adding these annotations will allow us to pull each resource from the parent command when running `resources`